doc/contributing/documentation_guide.md

```diff
@@ -328,7 +327,7 @@ Return types:
 
 Aliases:
 
-- Omit aliases from the `call-seq`, but mention them near the end (see below).
+- Omit aliases from the `call-seq`.
```

Comment:
I would clarify that in rare circumstances aliases can be listed in the call-seq.  @zverok provided an example for `String#-@` and `String#dedup`.  In this case (operator and normal method are aliases), it probably makes sense to list both.  In this case, the operator and method can both be listed.

Reply:
I'm not sure how to characterize the exceptional cases.  Would you please add something to the doc?